### PR TITLE
git submodule update

### PR DESCRIPTION
**Pull request by levBagryansky**
Fixed git submodule command

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ build/
 
 # ide stuff
 .idea
+cmake-build-debug
```
Comment (c71n93) [inline .gitignore:10]: Don't forget to leave an empty line at the end of file

File: Makefile [hunk 2]
```diff
@@ -11,7 +11,7 @@ all: init format build test
 
 .PHONY: init
 init:
-	git submodule init
+	git submodule update
```
Comment (c71n93) [inline Makefile:14]: You fist need to `git submodule init` and then `git submodule update` to clone repository as submodule

File: .gitignore (hunk 1 above)
Comment (levBagryansky) [reply .gitignore:10]: Why?

**Review by c71n93 — APPROVED** [review]

**Review by mmamayka — APPROVED** [review]
LGBT

**Pull request merged**